docs/user-guides/userguide_linux.md

```diff
@@ -57,45 +59,65 @@ The Mumble Server is either started [automatically](#auto-start) or [manually](#
 On many distributions the Mumble Server is started automatically on system boot.
 
 You can also disable this and start the Server manually.
-Most distributions use `systemd` now, so search in the systemd documentation on how to disable the Mumble Server service.
+Most distributions use `systemd` now, so search in the systemd documentation on how to disable the Mumble Server service (e.g. `murmur.service`).
+
+Next Steps: 
+
+1. Set the SuperUser (Server Admin) password:
+
+```
+murmurd -ini /etc/murmur.ini -supw PASSWORD
+```
+
+2. Restart the Server:
+If your system uses `systemd`, use e.g.:
+
+```
+systemctl restart murmur.service
+```
+
+3. Login & administrate the Server:
+See [Server Configuration Guide](server_config_guide.md#administration-with-mumble-client) for details.
 
 #### Manual Start
 
-The Mumble Server should be run from the command line, so start a terminal.
-(*If the `Static Linux Server` is installed or the binary is not linked, you have to change directory to wherever the Mumble Server is installed, first.*)
+The Mumble Server should be run from a terminal.
+(*If the `Static Linux Server` is installed or the binary is not linked, you have to change the directory to wherever the Mumble Server is installed, first.*)
 
-Run the Mumble Server with:
+<!-- Will the binary name change in 1.4? -->
 
-<!-- Step by Step guide?:
-1. Set Superuser password:
+<!-- mention different binary name of static linux server? -->
+
+1. Set the SuperUser (Server Admin) password:
 
 ```
-murmurd [-supw <password>]
+murmurd -supw <password>
 ```
 
-2. Serverstart
-Note: Flags are optional.
+2. Start the server:
+
+*Note:* Flags are optional (see below).
```

Comment:
Did you use italics instead of bold on purpose?

Reply:
Yes, I thought that for simple hints it would be enough to use italics and for important notes I use bold.
But it's up to you, if you don't like it, I will change it.

Reply:
Nah it's fine. I just think it's better to use single underscores for italics in order to make clear that italics is indeed the intentional choice :shrug: